onelive/archive/index.html

```diff
@@ -34,7 +34,7 @@
 
 <main>
     <!--main content goes here-->
-    <section style="background: url('../images/header-onelive.svg') right bottom no-repeat; height: 70vh">
+    <section style="background: url('../images/header-onelive.svg') right bottom no-repeat; height: 87vh">
```

Comment:
Let's create a CSS class for this

Reply:
Sure

Reply:
@anjula-sack I rechanged the code again. I think this implementation will be good. because all of our other pages use this implementation.
**Other pages normally use two sections for both Mobile and Bigger screen views.**

**ERROR**
![onlive text overlap](https://user-images.githubusercontent.com/88109775/203975761-a15b5855-6900-441b-8b6f-dbaa680a2ec6.png)


**After Fixing**
![finalize](https://user-images.githubusercontent.com/88109775/203975641-df466f9e-9027-431c-b991-e99a106a4c9c.png)
![finalize2](https://user-images.githubusercontent.com/88109775/203975664-b32f1a43-5e15-4867-84d0-45995f61720b.png)

Reply:
@anjula-sack is this requested changes done?